File: PRAK404_2510817210004_MuhammadRizkyramadhani.py
#Program Kalkulator Sederhana
print('Selamat datang di program Kalkulator Sederhana!')
print('_______________________________________________\n')

while True:
    print("Pilih program :")
    print("1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    operasi = int(input("Masukkan pilihanmu : "))
    print('_______________________________________________\n')
    nama_saya = ("Muhammad Rizky Ramadhani")

    if operasi == 1:
        nilai1 = int(input("Masukkan nilai pertama : "))
        nilai2 = int(input("Masukkan nilai kedua : "))
        print(f'Hasil Penjumlahan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1+nilai2:.2f}')
        print("\n🤔 Ada yang bisa saya bantu lagi❓\n")
    elif operasi == 2:
        nilai1 = int(input("Masukkan nilai pertama : "))
        nilai2 = int(input("Masukkan nilai kedua : "))
        print(f'Hasil Pengurangan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1-nilai2:.2f}')
        print("\n🤔 Ada yang bisa saya bantu lagi❓\n")
    elif operasi == 3:
        nilai1 = int(input("Masukkan nilai pertama : "))
        nilai2 = int(input("Masukkan nilai kedua : "))
        print(f'Hasil Perkalian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1*nilai2:.2f}')
        print("\n🤔 Ada yang bisa saya bantu lagi❓\n")
    elif operasi == 4:
        nilai1 = int(input("Masukkan nilai pertama : "))
        nilai2 = int(input("Masukkan nilai kedua : "))
        if nilai2 == 0:
            print("\n❗ Pembagian dengan nol tidak diperbolehkan. Silakan masukkan nilai selain nol.\n")
        else:
            print(f'Hasil Pembagian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1/nilai2:.2f}')
            print("\n🤔 Ada yang bisa saya bantu lagi❓\n")
    elif operasi == 5:
        print(f"Terimakasih, telah menggunakan kalkulator {nama_saya} 😊🙏")
        break
    else:
        print('❗ Input anda salah, silahkan coba lagi\n')

File: PRAK404_2510817210004_MuhammadRizkyramadhani.c
#include <stdio.h>

int main() {
    int operasi, nilai1, nilai2;
    char nama_saya[] = "Muhammad Rizky Ramadhani";

    printf("Selamat datang di program Kalkulator Sederhana!\n");
    printf("_______________________________________________\n\n");

    while (1) {
        printf("Pilih program :\n");
        printf("1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("Masukkan pilihanmu : ");
        scanf("%d", &operasi);
        printf("_______________________________________________\n\n");

        if (operasi == 1) {
            printf("Masukkan nilai pertama : ");
            scanf("%d", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%d", &nilai2);
            printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", 
                   (float)nilai1, (float)nilai2, (float)(nilai1 + nilai2));
            printf("\nAda yang bisa saya bantu lagi?\n\n");
        } else if (operasi == 2) {
            printf("Masukkan nilai pertama : ");
            scanf("%d", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%d", &nilai2);
            printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", 
                   (float)nilai1, (float)nilai2, (float)(nilai1 - nilai2));
            printf("\nAda yang bisa saya bantu lagi?\n\n");
        } else if (operasi == 3) {
            printf("Masukkan nilai pertama : ");
            scanf("%d", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%d", &nilai2);
            printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", 
                   (float)nilai1, (float)nilai2, (float)(nilai1 * nilai2));
            printf("\nAda yang bisa saya bantu lagi?\n\n");
        } else if (operasi == 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%d", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%d", &nilai2);
            if (nilai2 == 0) {
                printf("\nPembagian dengan nol tidak diperbolehkan. Silakan masukkan nilai selain nol.\n\n");
            } else {
                printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", 
                       (float)nilai1, (float)nilai2, (float)nilai1 / nilai2);
                printf("\nAda yang bisa saya bantu lagi?\n\n");
            }
        } else if (operasi == 5) {
            printf("Terimakasih, telah menggunakan kalkulator %s\n", nama_saya);
            break;
        } else {
            printf("Input anda salah, silahkan coba lagi!\n\n");
        }
    }

    return 0;
}